packages/web/hooks/use-convert-variant.ts

```
@@ -0,0 +1,213 @@
+                  },
+                ]);
+                resolve();
+              },
+              onBroadcastFailed: reject,
+            }
+          )
+          .catch((e) => {
+            console.error("Error broadcasting convert variant tx", e);
+            reject(e);
+          });
+      }),
```

Biome

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)